src/core/main.c

```diff
@@ -1973,7 +1973,7 @@ static int invoke_main_loop(
 
                         manager_send_reloading(m);
 
-                        log_info("Reloading.");
+                        log_info("Reloading...");
```

Comment:
Maybe, `…` is better?

Reply:
I avoided that intentionally, so that grepping for "Reloading." still works

Reply:
I see. OK.